src/app/resource-actions/resource-actions.component.ts

```diff
@@ -0,0 +1,86 @@
+import { Component, OnInit } from '@angular/core';
+import {ProjectService} from '../../services/project-service/project.service';
+import {Resource} from '../../interfaces/resource';
+import {ResourceService} from '../../services/resource-service/resource.service';
+import {ActivatedRoute, Params} from '@angular/router';
+import {EntityActionsComponent} from '../entity-actions/entity-actions.component';
+import {ToastrService} from 'ngx-toastr';
+import {Model} from '../../interfaces/model';
+
+@Component({
+  selector: 'app-resource',
+  templateUrl: './resource-actions.component.html',
+  styleUrls: ['./resource-actions.component.css']
+})
+export class ResourceActionsComponent extends EntityActionsComponent implements OnInit {
```

Comment:
Why is component inheritance necessary here?

Reply:
why not>?